package.json

```
@@ -10,7 +10,7 @@
         "table",
         "dragtable"
     ],
-    "version": "2.0.15",
+    "version": "2.0.16",
```

I wouldn't bump this but rather append to the end in a semver valid fashion so this can't ever be confused with an official version. Something like:
```suggestion
    "version": "2.0.15-isight1",
```